bin/fetch-configlet

```diff
@@ -1,16 +1,26 @@
 #!/usr/bin/env python3.7
+from datetime import datetime, timezone
 import io
-import tarfile
+import json
 import platform
-import requests
 import sys
+import tarfile
+import time
+from urllib.request import urlopen
 
 
 def download_and_extract(url):
-    resp = requests.get(url)
-    print('extracting...')
-    file = tarfile.open(fileobj=io.BytesIO(resp.content), mode='r:gz')
-    file.extractall(path='bin/')
+    with urlopen(url) as request:
+        status = request.status
+        if status >= 400:
+            print(f"Unexpected {status} status from download server.")
+            time.sleep(5)
+            return 1
+        response = request.read()
+    print("extracting...")
+    file = tarfile.open(fileobj=io.BytesIO(response), mode="r:gz")
```

Comment:
Shadows `file` builtin; I know this late in the function that's a minimal issue, but best practice is still to avoid shadowing.

Reply:
I am unfamiliar with the `file` built in you are refrencing? Perhaps you are thinking of `open` or the `__file__` property?

Reply:
Ahh, sorry, that’s an old habit from the days when we supported Python 2 dying slower than it should have... not a builtin any longer. Disregard.